zapetw/core_windows.go

```diff
@@ -0,0 +1,39 @@
+package zapetw
+
+import (
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func getETWencoder() zapcore.Encoder {
+	encoderConfig := zap.NewProductionEncoderConfig()
+	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	return zapcore.NewJSONEncoder(encoderConfig)
+}
+
```

Comment:
similar comment - this is standard zap encoder code, we don't need to do it and can let the caller do it